events/EventQueue.h

```diff
@@ -18,14 +18,16 @@
 #ifndef EVENT_QUEUE_H
 #define EVENT_QUEUE_H
 
-#include "equeue/equeue.h"
+#include "events/source/equeue.h"
```

Comment:
equeue.h should be in internal/

Reply:
I thought we were not going for `internal` dirs but instead will only have `source`. When did this change?

Reply:
91ef36dd78